README.md

```diff
@@ -113,15 +113,28 @@ If you are an Enterprise user with first-party hosting access, you will need to
 You can then enable it in your code:
 
 ``` swift
-public enum Endpoint {
-    case default, alternate
-}
+let hcaptcha = try? HCaptcha(
+    ...
+    endpointApi: URL("https://custom.api.endpoint")!,
```

Comment:
```suggestion
    jsSrc: URL("https://js.hcaptcha.com/1/api.js")!,
```

.. but this should be optional, the default is correct for anyone not using first-party hosting.